### DISCUSSION TITLE
libpag.so全局变量析构函数中JNI调用

### DISCUSSION THREAD

**Discussion by DXBbeginner** [Tencent/libpag#2351] [Q&A]
![70e9fce72c4c5f019b51730b8](https://github.com/Tencent/libpag/assets/57246337/5d5bec1d-3383-4e94-af60-a2c3a27617eb)
libpag.so看上去会依赖于在全局变量的析构函数里attach jvm做JNI调用，这个具体是什么场景呢，执行的是什么相关操作么，是不是不太合理。

**Comment by kevingpqi123** [comment]
反馈问题请提供完整的信息：如复现问题的版本号、完整的堆栈信息（符号解析后的），只有大家提供的信息越全面你们才能获取到更完整的分析

**Reply by DXBbeginner** [reply]
版本是4.3.33 堆栈信息是完整的，请问一下 需要怎么解析呢

**Reply by kevingpqi123** [reply]
可以看下这篇文章：https://juejin.cn/post/6844903761006379015?from=search-suggest ，这些都是通用的解析方法
关于包含完整符号的文件，我们在 github/release 的发布版本中有提供对应的 obj.zip 文件

**Comment by CodeJhF** [comment]
之前请你补充的复现信息（demo / 版本号 / 机型 / 日志）一直没有收到，缺少这些信息无法继续定位，先关闭本条。
如果问题还在，欢迎补齐上述信息后回复或重开，我们会优先跟进。（stale-close）
